app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -1,17 +1,18 @@
 <%
-  links_hash = content_item.with_indifferent_access["links"] || {}
+
+  links_hash = content_item.to_h[:links] || {}
 
   organisations = []
-  organisations += links_hash["organisations"] || []
-  organisations += links_hash["lead_organisations"] || []
-  organisations += links_hash["supporting_organisations"] || []
-  organisations += links_hash["worldwide_organisations"] || []
+  organisations += links_hash[:organisations] || []
```

Comment:
It is, but component templates _try_ to avoid using too many ruby/ERB specific features, with the idea that they could be translated into something like mustache and used elsewhere easily - though this template is very much the exception to that rule :(

I think the key-value-or-empty-array behaviour could be better expressed using `slice` anyway, so I'm inclined to leave that, and this, kind of refactoring to another PR, and this one can fix the live issue.